src/components/Carousel/index.tsx

```diff
@@ -5,9 +5,24 @@ import styled from '@emotion/styled'
 import { useMediaQuery } from '@hooks'
 
 export interface CarouselProps {
+  /** Carousel 컴포넌트에 들어갈 이미지들을 정합니다.
+   * @type { url: string, id: number } []
+   */
   images: { url: string; id: number }[]
+
+  /** Carousel 컴포넌트에 화살표의 유무를 정합니다.
+   * @type boolean
+   */
   isArrow: boolean
+
+  /** Carousel 컴포넌트에 들어갈 이미지들의 크기를 정합니다.
+   * @type number
```

Comment:
```suggestion
   * @type number | undefined
```
undefined 필요할 것 같아요!